src/Build/BackEnd/BuildManager/BuildParameters.cs

```diff
@@ -351,78 +330,69 @@ public ThreadPriority BuildThreadPriority
         /// </summary>
         public bool UseSynchronousLogging
         {
-            get { return _useSynchronousLogging; }
-            set { _useSynchronousLogging = value; }
+            get => _useSynchronousLogging;
+            set => _useSynchronousLogging = value;
         }
 
         /// <summary>
         /// Gets the environment variables which were set when this build was created.
         /// </summary>
-        public IDictionary<string, string> BuildProcessEnvironment
-        {
-            get
-            {
-                if (_buildProcessEnvironment == null)
-                {
-                    return new ReadOnlyDictionary<string, string>(new Dictionary<string, string>(0));
-                }
-
-                return new ReadOnlyDictionary<string, string>(_buildProcessEnvironment);
-            }
-        }
+        public IDictionary<string, string> BuildProcessEnvironment => _buildProcessEnvironment == null
+            ? new ReadOnlyDictionary<string, string>(new Dictionary<string, string>(0))
+            : new ReadOnlyDictionary<string, string>(_buildProcessEnvironment);
```

Comment:
Also, this seems confusing, like it should have been a lazy over `_buildProcessEnvironment` but isn't?